app/views/lists/show.html.erb

```diff
@@ -0,0 +1,79 @@
+<%= render 'shared/breadcrumbs', {
+  links: [
+    {
+      text: @tag.is_a?(MainstreamBrowsePage) ? "Mainstream browse pages" : "Topics",
+      href: @tag.is_a?(MainstreamBrowsePage) ? mainstream_browse_pages_url : topics_url
+    },
+    *([
+      text: @tag.parent[:title],
+      href: polymorphic_path(@tag.parent)
+    ] if @tag.parent.present? ),
+    {
+      text: @tag[:title],
+      href: polymorphic_path(@tag),
+    },
+    {
+      text: "List",
+    }
+  ]
+} %>
+
+<%= content_for :context, "List" %>
+
+<% content_for :title, @list.name %>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
```

Comment:
Is there a reason why we have `one-half` inside a `two-thirds`?

Reply:
I think i was probably just trying to align them right correctly iirc. I can probs just float them right though. Let me have a quick look to remind myself.